polaris.shopify.com/src/components/PatternPage/SingleVariantPatternPage.tsx

```diff
@@ -0,0 +1,121 @@
+import React, {useState} from 'react';
```

Comment:
Let's refactor this to be a single PatternPage component again, such that the PatternPage takes on the logic of rendering based on whether a pattern variant exists or not.